tests/chart_tests/test_prometheus_statefulset.py

```diff
@@ -127,3 +127,26 @@ def test_prometheus_with_multiple_extraFlags(self, kube_version):
             "--enable-feature=remote-write-receiver" in c_by_name["prometheus"]["args"]
         )
         assert "--enable-feature=agent" in c_by_name["prometheus"]["args"]
+
```

Comment:
Prometheus is missing the `"persistentVolumeClaimRetentionPolicy" not in doc["spec"]` test case